prime-router/src/main/kotlin/cli/TestReportStream.kt

```diff
@@ -1402,17 +1402,20 @@ class InternationalContent : CoolTest() {
     }
 }
 
+/**
+ * Creates fake data as if from a sender and tries to send it to every state and territory
+ */
 class SantaClaus : CoolTest() {
 
     override val name = "santaclaus"
     override val description = "Creates fake data as if from a sender and tries to send it to every state and territory"
-    override val status = TestStatus.GOODSTUFF
+    override val status = TestStatus.DRAFT
 
     override fun run(environment: ReportStreamEnv, options: CoolTestOptions): Boolean {
 
         var passed = true
 
-        if (options.env == "prod") {
+        if (options.env in listOf("prod", "test")) {
```

Comment:
This is still not right.   The requirement is  "_test that the env is local or staging only. If not local or staging, ABORT the test."_

So the test should be:

if (env _not in_ list(local, staging)) {
    return the error;
}

Again, here's the bug:  suppose in the future we create some new production environment "prod2".  The way your code is written, the test will run in that environment, which is bad.